src/webgl/interaction.js

```diff
@@ -91,14 +105,81 @@ p5.prototype.orbitControl = function(sensitivityX, sensitivityY, sensitivityZ) {
     this._setProperty('wheelDefaultDisabled', true);
   }
 
-  const zoomScaleFactor = 0.02;
+  // disable default touch behavior on the canvas element and add
+  // 'touchActionsDisabled' flag to p5 instance
+  const { disableTouchActions = true } = options;
+  if (this.touchActionsDisabled !== true && disableTouchActions) {
+    this.canvas.style['touch-action'] = 'none';
+    this._setProperty('touchActionsDisabled', true);
+  }
+
+  // get moved touches
+  const movedTouches = this.touches.filter(t => t.moved);
+  // flags for interaction
+  let zoomFlag = false;
```

Comment:
Maybe we can rename these flag variables to something like `shouldZoom` so it's clearer what will happen when they're true or false?

Reply:
Writing ~Flag is certainly not appropriate...
Well, what about executeZoomProcess, executeRotateProcess, executeMoveProcess?
I think it's easy to understand in the sense that appropriate processing is executed when true...